.editorconfig

```diff
@@ -176,14 +176,11 @@ end_of_line = crlf
 # Code files
 [*.{cs,vb}]
 
-dotnet_diagnostic.UseAsyncSuffix.severity = error
+dotnet_diagnostic.VSTHRD200.severity = error
 
-dotnet_diagnostic.UseConfigureAwait.severity = error
-
-dotnet_diagnostic.AvoidAsyncSuffix.severity = none
-
-dotnet_diagnostic.AvoidAsyncVoid.severity = none
+dotnet_diagnostic.VSTHRD111.severity = error
 
+dotnet_diagnostic.VSTHRD100.severity = none
```

Comment:
So these diagnostics replace the old ones. Would be to see if we can enable all the other diagnostics from this analyzer as well (as a separate PR).

Reply:
@bgavrilMS makes sense. Here is the list of the other analyzers in this package:
https://github.com/microsoft/vs-threading/blob/main/doc/analyzers/index.md

Definitely a good idea to enable the critical ones at least